app/models/session.rb

```diff
@@ -7,19 +7,29 @@ class Session < Common::RedisStore
   redis_key :token
 
   DEFAULT_TOKEN_LENGTH = 40
+  MAX_SESSION_LIFETIME = 12.hours
 
   attribute :token
   attribute :uuid
+  attribute :created_at
 
   validates :token, presence: true
-  # validates other attributes?
+  validates :uuid, presence: true
```

Comment:
Unrelated to Session extension, but `Session.uuid` should never be nil.  Boy scout rule